test/e2e/framework/util.go

```diff
@@ -326,11 +327,8 @@ func InstallCluster(t TestingTInterface, ctx context.Context, source, server Run
 					continue
 				}
 				var ready bool
-				for _, condition := range updated.Status.Conditions {
-					if condition.Type == clusterv1alpha1.ClusterConditionReady && condition.Status == corev1.ConditionTrue {
-						ready = true
-						break
-					}
+				if conditions.IsTrue(updated, clusterv1alpha1.ClusterReadyCondition) {
```

Comment:
Now that there is no longer a loop, maybe just test the conditional directly for indication of success?

```golang
if conditions.IsTrue(updated, clusterv1alpha1.ClusterReadyCondition) {
  return nil
}
```

Reply:
👍🏼  Thanks.